components/02-form-elements/typeahead/typeahead.config.js

```diff
@@ -0,0 +1,201 @@
+import { orderBy } from 'lodash';
+import { occupations } from './occupations';
+import { ethnicities } from './ethnicities';
+
+const territoriesJSON = {
```

Comment:
**Query**
I thought that this was going to be using the .gov country register as this is the canonical source for gov work?
https://www.registers.service.gov.uk/registers/country

Reply:
cc @charlotte-hirst

Reply:
This is dummy data I've used because I wanted Welsh country names to test some functionality and I couldn't fine a Welsh version on the register. I wouldn't recommend using this data in production.